src/Styleguide/Components/Stepper.tsx

```diff
@@ -7,36 +7,39 @@ import { Flex } from "Styleguide/Elements/Flex"
 interface StepperProps extends TabsProps {
   initialTabIndex: number
 
-  /* the step user currently is at */
+  /* the step user currently is at (e.g. previous steps completed) */
   currentStepIndex: number
+
+  disableNavigation?: boolean
 }
 
 const transformTabBtn = (
   element: JSX.Element,
   tabIndex: number,
   props: any
 ): JSX.Element => {
-  const { currentStepIndex } = props
+  const { currentStepIndex, disableNavigation } = props
+  const elementWithoutNav = React.cloneElement(element, {
+    ...element.props,
+    onClick: () => {
+      null
+    },
```

Comment:
So one thing that might be nice for a disabled nav element, is injecting an additional style prop e.g. `style: { cursor: "default" }` to remove the click affordance that is still currently there because of the cursor pointer.

Reply:
haha turned out adding that insignificant `cursor: default` required some overhaul (https://github.com/artsy/reaction/pull/1049/commits/11a03b7e0bdb2f50e364a95d1d12f1eee6a01855) but nice catch thanks.